CONTRIBUTING.md

```diff
@@ -71,7 +71,18 @@ Note: One or more pull requests must be submitted addressing all user story goal
 
 ## Communications mechanisms
 
-You can contact the maintainers of this project at [maintainers@metaschema.dev](mailto:maintainers@metaschema.dev) if you are interested in contributing to the development of this project or exchanging ideas. 
+You can contact the maintainers of this project at [maintainers@metaschema.dev](mailto:maintainers@metaschema.dev) if you are interested in contributing to the development of this project or exchanging ideas.
+
+## Developer information
+
+### Core metaschema functions and custom project functions
+
+The Metaschema [specification](https://pages.nist.gov/metaschema/specification/syntax/metapath/) and the conformant [metaschema-java](https://github.com/metaschema-framework/metaschema-java) library implement the Metapath functions required by the specification. However, other libraries, such as this [this project's library](/) can extend the inventory of available functionality with custom functions that are not required in the core specification. These are OSCAL-specific functions, not core Metaschema functions. At the the of the v4.1.0 release, there are only two custom OSCAL-specific functions in this library.
```

Comment:
@david-waltermire, once https://github.com/metaschema-framework/metaschema-java/pull/124 is merged, can we update the URL to where you want it if you expect it into main eventually? I don't want to break link checking, so I won't start out with that.

```suggestion
The Metaschema [specification](https://pages.nist.gov/metaschema/specification/syntax/metapath/) and the conformant [metaschema-java](https://github.com/metaschema-framework/metaschema-java/tree/develop/CONTRIBUTING.md#core-metaschema-functions-functions) library implement the Metapath functions required by the specification. However, other libraries, such as this [this project's library](/) can extend the inventory of available functionality with custom functions that are not required in the core specification. These are OSCAL-specific functions, not core Metaschema functions. At the the of the v4.1.0 release, there are only two custom OSCAL-specific functions in this library.
```

Reply:
Note the `[this project's library](/)` still looks not right.

Reply:
Yeah I spent a few minutes trying different alternatives and none seem to work, I will just add the full URL.

```suggestion
The Metaschema [specification](https://pages.nist.gov/metaschema/specification/syntax/metapath/) and the conformant [metaschema-java](https://github.com/metaschema-framework/metaschema-java) library implement the Metapath functions required by the specification. However, other libraries, such as this [this project's library](https://github.com/metaschema-framework/liboscal-java) can extend the inventory of available functionality with custom functions that are not required in the core specification. These are OSCAL-specific functions, not core Metaschema functions. At the the of the v4.1.0 release, there are only two custom OSCAL-specific functions in this library.
```